hardware/simulation/icarus/Makefile

```diff
@@ -1,8 +1,8 @@
 #module paths
-CACHE_DIR:=../../..
-INTERCON_DIR:=$(CACHE_DIR)/submodules/iob-interconnect
-MEM_DIR:=$(CACHE_DIR)/submodules/iob-mem
-AXI_MEM_DIR:=$(CACHE_DIR)/submodules/axi-mem
+ROOT_DIR:=../../..
+INTERCON_DIR:=$(ROOT_DIR)/submodules/iob-interconnect
```

Comment:
useless repetition, see?

Reply:
Sure thing, everything fixed now. Sorry for the trouble.